### PR TITLE
Add sirius

### PR DESCRIPTION
**Pull request by solidv**

### CODE REVIEW

File: _data/devices/sirius.yml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+- Dual SIM
+- Fingerprint reader
+- Accelerometer
+- Barometer
+- Gyroscope
+- Hall effect sensor
+- Light sensor
+- Proximity sensor
+- Compass
+- IR
```
Comment (jhenrique09) [inline _data/devices/sirius.yml:49]: By IR you mean IR blaster? if yes, change it

Comment (Hlcpereira) [reply _data/devices/sirius.yml:49]: He means IR for faceunlock. Sirius also has it so does beryllium and dipper

Comment (jhenrique09) [reply _data/devices/sirius.yml:49]: so it's IR camera

Comment (solidv) [reply _data/devices/sirius.yml]: We dont have IR camera, we have IR blaster

**Comment by jhenrique09** [comment]
One last thing, your device uses AOSP recovery?

**Comment by solidv** [comment]
No, we are using twrp. Going to add

**Comment by solidv** [comment]
Noticed twrp in wiki is not correct, in twrp we have the codename xmsirius instead of sirius

**Pull request merged**